ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -1085,8 +1085,7 @@ presubmits:
           /usr/bin/promoted-image-governor --ci-operator-config-path=ci-operator/config --release-controller-mirror-config-dir=core-services/release-controller/_releases --openshift-mapping-dir=core-services/image-mirroring/openshift --openshift-mapping-config=core-services/image-mirroring/openshift/_config.yaml --dry-run=true
           git add -N .
           if ! git diff --exit-code; then
-            >&2 echo "run 'make openshift-image-mirror-mappings' if the promote tags to the ocp or origin namespaces are changed in the pull request"
-            git diff
```

Comment:
The diff would be shown in the output of `! git diff --exit-code` if `! git diff --exit-code` failed.